### ISSUE TITLE
Comments

### ISSUE THREAD

**Issue by ghost** [UBC-MDS/525_group10#15]
File names in your Github repository could be improved.
The notebook needs to be better organized: you could have kept the rubric from the wording.
It is hard to follow what's being done as there are very few comments on the code, and very little analysis of your observations/results.
Comparison of performances between the different computers of your group?

**Comment by dbandrews** [comment]
Thanks Paul - We'll taking these into consideration in future milestones!
